Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -9,13 +9,14 @@ var fileEncoding = require('./fileencoding.js');
 
 export async function substituteAppSettingsVariables(folderPath) {
     var configFiles = tl.glob(folderPath + "/**/*.config");
-    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections"];
+    var variableMap = varUtility.getVariableMap();
+    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections", "system.webServer"];
```

Comment:
After adding system.weServer  in tag list, it start looking this tag for substitution, if tag not found it will skip it. So verify if this tag can exist in asp.net4 and asp.net core webapps. If that tag is only for python app its completely safe.